### ISSUE TITLE
total training steps and tokenization in run_glue

### ISSUE THREAD

**Issue by xinsu626** [huggingface/pytorch-transformers#957]
Question for total Training Steps: 

In run_glue line 78, the total number of training steps is calculated using `t_total = len(train_dataloader) // args.gradient_accumulation_steps * args.num_train_epochs`.

I was wondering if we use gradient accumulation the num_train_epochs in the above code is not actual training epochs instead the actual training epochs may be `args.num_train_epochs/args.gradient_accumulation_steps`. So the total training steps should be `t_total = (len(train_dataloader) // args.gradient_accumulation_steps) * (args.num_train_epochs / args.gradient_accumulation_steps)`. Is my understanding correct?

Question for tokenization:

I saw in your `utils_glue.py`'s `convert_examples_to_features` function, you set `cls_token_at_end=False, pad_on_left=False` , but didn't provide any accesses to change these parameters when users want to fine tune xlnet. Does this will decrease the xlnet fine tuning performance? 

Thank you.
